filebeat/magefile.go

```diff
@@ -220,3 +224,57 @@ func PythonIntegTest(ctx context.Context) error {
 		return devtools.PythonNoseTest(args)
 	}, "GENERATE", "TESTING_FILEBEAT_MODULES", "TESTING_FILEBEAT_FILESETS")
 }
+
```

Comment:
Try putting this into `filebeat/scripts/mage/` so that you don't have to duplicate it.

Reply:
You could put them under `filebeat/scripts/mage/generate`. Then import them with `// mage:import generate` so you get targets like `mage generate:module`.